src/routes/_layout.svelte

```diff
@@ -25,6 +25,11 @@
   $: path = $page.path;
 </script>
 
+<svelte:head>
+  <meta property="og:type" content="website" />
+  <meta name="twitter:domain" value="https://h4iuiuc.netlify.app" />
```

Comment:
This is fine for now, but we have to be diligent about replacing this with our actual domain. I'd argue we just do that now.

Not leaving a comment for every time this happens, just code search for `h4iuiuc.netlify.app` when replacing